context/auth.tsx

```diff
@@ -33,6 +32,13 @@ export const AuthProvider = ({ children }: ProviderProps) => {
         setUser(JSON.parse(data));
       }
     });
+    /**
+     * Reason: The `useAsyncStorage` "experimental" hook is not memoized and always returns new functions
+     * which causes an infinite loop when `getItem` is used as a dependency in `useEffect`.
```

Comment:
Didn't know about this.

Reply:
I knew this was experimental but didn't know about the looping.